pkg/aggregate/aggregate.go

```diff
@@ -223,3 +232,140 @@ func (a AggregatesList) Clear() {
 		aggr.Clear()
 	}
 }
+
+func GetHiddenAggregates(mask AggrType, requestedAggregates []AggrType) []AggrType {
+	var hiddenAggregates []AggrType
+
+	for _, aggr := range rawAggregates {
+		if aggr&mask == aggr && !ContainsAggregate(requestedAggregates, aggr) {
+			hiddenAggregates = append(hiddenAggregates, aggr)
+		}
+	}
+	return hiddenAggregates
+}
+
+func GetHiddenAggregatesWithCount(mask AggrType, requestedAggregates []AggrType) []AggrType {
+	mask |= aggrTypeCount
+	return GetHiddenAggregates(mask, requestedAggregates)
+}
+
+func ContainsAggregate(list []AggrType, item AggrType) bool {
```

Comment:
see my previous comment, you can make it much more efficient with bitwise operations